code/media/src/main/java/com/adobe/marketing/mobile/media/internal/MediaExtension.java

```diff
@@ -30,11 +33,10 @@ public class MediaExtension extends Extension {
 
     final Map<String, MediaTrackerInterface> trackers;
     MediaState mediaState;
-    MediaRealTimeService mediaRealTimeService;
-    MediaOfflineService mediaOfflineService;
 
```

Comment:
Let's add TODO to add/init MediaEventProcessor

Reply:
I'm adding MediaEventProcessor in the next PR so we don't need the TODO here. I already have the code locally.